lib/model/QubitInformationObject.php

```diff
@@ -437,6 +437,8 @@ public static function getUpdatedRecords($options = [])
         }
 
         $criteria->addAscendingOrderByColumn(QubitObject::UPDATED_AT);
+	// Secondarily filter by ID in case UPDATED_AT has duplicate entries.
```

Comment:
Hi @rferris! Thanks so much for your contribution! I just finished looking at your changes and this definitely makes sense to me, and should fix the issue!

I have a few super quick changes I'd like to request before we can accept this PR —
1. Could you rebase this to the branch `qa/2.x` instead of `stable/2.7`. We use that branch for any bug fixes or features intended for the next release, and as you might've noticed here there's a few extra changes it picked up that we don't need, when we changed the base to `qa/2.x` here on the PR.
2. Line `#440` seems to be using tabs for indentation which is making PHP Coding Standards fixer throw an error in the tests. Could you make sure this uses spaces instead?
3. Small nitpick, but this repo uses uppercase first letter for the commit message.

Let us know if you have the time to make these changes. If not, we're happy to do these and accept this fix. Thanks again!

Reply:
Hi Anvit,

Can do!

Rohan

Reply:
Awesome! Thanks Rohan! Really appreciate it!

Reply:
Closed this PR and reissued over at https://github.com/artefactual/atom/pull/1593